src/tbbmalloc/tbbmalloc_internal.h

```diff
@@ -235,6 +235,8 @@ class BitMaskMax : public BitMaskBasic<NUM> {
         BitMaskBasic<NUM>::set(NUM - 1 - idx, val);
```

Comment:
For the reason of symmetry, is it making sense to have similar assert in the `set()` as well? Because now a random reader may ask "Why only in `getMaxTrue()`"?

Reply:
Done